packages/rancher-vsphere-csi/package.yaml

```diff
@@ -1,3 +1,2 @@
 url: local
-packageVersion: 0
-doNotRelease: true
\ No newline at end of file
+packageVersion: 0
```

Comment:
we should bump this to 1 so we make a change that isn't modifying an already released chart

Reply:
@aiyengar2 I bumped the packageVerison as you mentioned. Can you please review it again?